pkg/labeller/labeller.go

```diff
@@ -16,39 +15,79 @@ type FetchableObject struct {
 }
 
 type Labeller struct {
-	APIProxy       apiproxy.APIProxy
-	ObjectMeta     *metav1.ObjectMeta
-	ComponentLabel string
-	ComponentName  string
-	Annotations    map[string]string
+	ObjectMeta                 *metav1.ObjectMeta
+	ComponentObjectsNamePrefix string
+	ComponentFullName          string
+	annotations                map[string]string
+
+	instanceName string
+
+	objectsNamePrefixBase string
```

Comment:
Let's add description for this fields here with examples so it would be obvious to understand what values to expect in those fields for singleton and multiinstance components.

For now it is hard to say what will be in values and no tests to see what they are.